radar.csv

```diff
@@ -1,8 +1,8 @@
-﻿name,ring,quadrant,isNew,description
+name,ring,quadrant,isNew,description
 "C#",adopt,"Languages and Frameworks",true,"<a href='https://docs.microsoft.com/en-us/dotnet/csharp/'>C#</a> is our language of choice. It's strongly-typed, blends functional and object-oriented programming and works on all the OS we need to support."
 "TypeScript",adopt,"Languages and Frameworks",true,"<a href='https://www.typescriptlang.org/'>TypeScript</a> is our default choice for running code in the browser. It's strongly-typed and smooths out the rough bits of JavaScript."
 "Go",trial,"Languages and Frameworks",true,"<a href='https://golang.org/'>Go</a> is the natural language to use when extending Kubernetes with operators."
-"Java",trial,"Languages and Frameworks",true,"Redgate purchased Flyway, written in <a href='https://www.oracle.com/java/'>Java</a>. Java is a plausible choice for new products (due to its rich ecosystem), but we default to C#."
+"Java 8",adopt,"Languages and Frameworks",true,"Redgate purchased Flyway, written in <a href='https://www.oracle.com/java/'>Java</a>. Java is a plausible choice for new products (due to its rich ecosystem), but we default to C#."
```

Comment:
Is Java 8 not now deprecated/eol?

Reply:
Sadly for Java 8 that's not the case. Java 8 has extended support until 2030. Unfortunately, due to changing in Oracle business strategies at the time they set their paid for extended support for Java 8 till 2030, which a lot of companies adopted, which outlasts even the latest LTS version of Java (17) which will only get extended support till 2029. This makes moving forward from Java 8 very difficult in the commercial space.

Reply:
In regards to your march 2022 date,, that's free updates. And in regards to the security only be 3 more years, I do not know where that has come from. 
Please see https://www.oracle.com/java/technologies/java-se-support-roadmap.html

Reply:
Fair enough - thanks for clarifying.